src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListComponents.tsx

```diff
@@ -128,14 +121,17 @@ export const AssigneeEdit = (params: GridRenderEditCellParams) => {
   return (
     <Autocomplete
       fullWidth
-      isOptionEqualToValue={(option, value) => option.id === value.id}
+      isOptionEqualToValue={(option, value) => {
```

Comment:
remove console.log/ change it back to how it was before since that syntax is cleaner